baseball/src/App.test.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {cleanup, render, fireEvent} from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+describe('Baseball', () => {
+  afterEach(cleanup);
+
+  describe('BUTTON FUNCTIONS', () => {
+    describe('Strike Button', () => {
+      it('should increase strikes display by 1', () => {
+        const {getByText} = render(<App/>);
+        expect(getByText(/strikes: 0/i));
```

Comment:
this is technically already handled by how `getByText` works on it's own, since it will throw a test fail if it can't the matching text, and will pass a success case if it does. definitely more explicit though